README.md

```diff
@@ -50,40 +50,47 @@ To switch back to the default binaries delete `JULIA_SCS_LIBRARY_PATH` and call
 
 ## Usage
 
-### MathOptInterface wrapper
+### High-level interfaces
 SCS implements the solver-independent [MathOptInterface](https://github.com/JuliaOpt/MathOptInterface.jl) interface, and so can be used within modeling softwares like [Convex](https://github.com/JuliaOpt/Convex.jl) and [JuMP](https://github.com/JuliaOpt/JuMP.jl). The optimizer constructor is `SCS.Optimizer`.
 
+A legacy [MathProgBase](https://github.com/JuliaOpt/MathProgBase.jl) interface is available as well, in maintanence mode only.
+
 ### Options
-All SCS solver options can be set through the direct interface(documented below) and through MathProgBase.
-The list of options is defined the [`scs.h` header](https://github.com/cvxgrp/scs/blob/58e9af926fabc6674a9f488d4e9761a4f0fc451c/include/scs.h#L43).
-To use these settings you can either pass them as keyword arguments to `SCS_solve` (high level interface) or as arguments to the `SCSSolver` constructor (MathProgBase interface), e.g.
+All SCS solver options can be set through the direct interface(documented below), through `Convex.jl` or `MathOptInterface.jl`.
+The list of options follows the [`glbopts.h` header](https://github.com/cvxgrp/scs/blob/0fd7ea85e8b0d878cacf5b1dbce557b330422ff7/include/glbopts.h#L30) in lowercase.
+To use these settings you can either pass them as keyword arguments to `SCS_solve` (high level interface) or as arguments to the `SCS.Optimizer` constructor (MathOptInterface), e.g.
 ```julia
 # Direct
 solution = SCS_solve(m, n, A, ..., psize; max_iters=10, verbose=0);
-# MathProgBase (with Convex)
-m = solve!(problem, SCSSolver(max_iters=10, verbose=0))
+# via MathOptInterface:
+solver = SCS.Optimizer(max_iters=10, verbose=0)
```

Comment:
This syntax is deprecated. We want to drop keyword argument to the constructor in the future.
It should rather be
```julia
optimizer = SCS.Optimizer()
MOI.set(optimizer, MOI.RawParameter("max_iters"), 10)
MOI.set(optimizer, MOI.RawParameter("verbose"), 0)
```